installer/src/main/bin/discoverDomain.sh

```diff
@@ -71,6 +103,12 @@ usage() {
   echo "        model_file      - the location of the model file to use,"
   echo "                          the default is to get the model from the archive"
   echo ""
+  echo "        variable_file    - the location of the variable file to write ,"
```

Comment:
The extra commas at the end are showing up in the script output.
The hyphen and text is indented one extra space.

Reply:
fixed thank you